pyproject.toml

```
@@ -31,13 +31,13 @@ include = ["README.md", "LICENSE"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-attrs = ">=20.1.0"
+attrs = ">=20.2.0"
 immutables = ">=0.20,<1.0.0"
 prompt-toolkit = "^3.0.0"
 pyrsistent = ">=0.18.0,<1.0.0"
 python-dateutil = "^2.8.1"
 readerwriterlock = "^1.0.8"
-typing_extensions = "^4.9.0"
+typing-extensions = "^4.9.0"
```

```suggestion
typing-extensions = "^4.7.0"
```

Do you want to do this or not? If it passes, then we can include it in this PR.

---

I will do as a follow up to this patch as suggest, but I'm currently investigating a couple of issues with using poetry to source the dependencies